frontend/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM node:20-slim
+FROM oven/bun:latest
```

Comment:
Have we tried this with 'oven/bun:slim' ? I haven't done a comparison 1:1 yet with node:20-slim vs bun:slim - but optimally I'd prefer to run the leanest base image we can (fewer dependencies, etc).